### ISSUE TITLE
Suppression Super Admin

### ISSUE THREAD

**Issue by Drakone** [pixelhumain/co2#481]
Est-ce qu'on pourrait donner le droit aux Super Admin de demander la suppression de n'importe quel élément ou au moins ceux qui dorment depuis quelques temps.

C'est frustrant de ne pas pouvoir supprimer ce genre d'orga : https://www.communecter.org/#openatlas

**Comment by clement59** [comment]
Normalement c'est bon pour les supers Admin tu as accés à la fonction directement

**Comment by Drakone** [comment]
Ah je pensai plutôt aux Admin Public.

**Comment by clement59** [comment]
Yes bonne question !! J'ai retravaillé le panel admin mais je sais pas si la supprssion marche pour toi (adminPublic) !
Et je sais pas si la fonction est bien fonctionnelle next release je pense
![capture d ecran de 2018-02-23 16-38-51](https://user-images.githubusercontent.com/6576514/36602331-0fe0e394-18b8-11e8-945d-8cdc546dfc14.png)
![capture d ecran de 2018-02-23 16-38-45](https://user-images.githubusercontent.com/6576514/36602332-1005e55e-18b8-11e8-8b2a-dcd16e455ff1.png)

**Comment by Drakone** [comment]
En tant qu'Admin Public j'ai pas accès au Directory.

**Comment by Drakone** [comment]
Je relance la demande. @clement59
